SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Supported Versions
+
+We do not support any old versions, all updates for bugs/features/security 
+will be released in next planned version.
+
+| Version      | Supported          |
+| ------------ | ------------------ |
+| LATEST only  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+We have mail-list
+https://checkstyle.org/mailing-lists.html
+you can find admin emails in it.
+you can find my email in my commits, you can clone our repo and see it by "git log".
+
+If we are not responding, please keep pining us, as final ersort create issue at
+https://github.com/checkstyle/checkstyle/issues with note that you have security issue
+
+if the vulnerability is accepted we create issue (without much details) on Github Issue tracker
+and we put label "approved" on it.
+if the vulnerability is declined, we can keep it private
+or if you would like to keep record of it issue tracker, we can do this also.
```

Comment:
`of it issue tracker` => `of it in issue tracker`

`we can do this also` => `you can do this also` ???